yal/optout.py

```diff
@@ -0,0 +1,276 @@
+                            f"- *{old_details['gender']}*",
+                            "",
+                            "*------*",
+                            "*Reply:*",
+                            "*1* - to see your personal data",
+                            "*0* - 🏠Back to Main *MENU*",
+                            "*#* - 🆘Get *HELP*",
+                        ]
+                    )
+                ),
+                helper_metadata={
+                    "image": contentrepo.get_image_url("bwise_header.png")
+                },
+            )
+        )
+
+        return await self.go_to_state(ChangePreferencesApplication.START_STATE)
+
+    async def state_tell_us_more(self):
+        return FreeText(
+            self,
+            question=self._(
+                "\n".join(
+                    [
+                        "🛑 STOP MESSAGING ME",
+                        "*Please tell us more*",
+                        "------",
+                        "",
+                        "🙍🏾‍♀️ *Thanks.*" "",
+                        "If you could share your reason by replying with a ",
+                        "few words about why you want to stop receiving messages,",
+                        "I'd be so grateful 🙂.",
+                    ]
+                )
+            ),
+            next="state_farewell_optout",
+        )
+
+    async def state_farewell_optout(self):
+        await self.worker.publish_message(
+            self.inbound.reply(
+                self._(
+                    "\n".join(
+                        [
+                            "🛑 STOP MESSAGING ME",
+                            "*Goodbye* 👋🏾",
+                            "-",
+                            "",
+                            "🙍🏾‍♀️Thanks so much for your help.",
+                            "You won't get any more messages from us.",
+                            "For any medical issues,",
+                            "please visit your nearest clinic.",
+                            "",
+                            "You're also welcome to visit us online, any time. 🙂👇🏾",
+                            "",
+                            "*Have a lovely day!* ☀️",
+                        ]
+                    )
+                )
+            )
+        )
+        # await asyncio.sleep(0.5)
+        return EndState(
+            self,
+            text=self._(
+                "\n".join(
+                    [
+                        "Need quick answers?",
+                        "Check out B-Wise online!👆🏾",
+                        "",
+                        "https://bwisehealth.com/",
+                        "",
+                        "You'll find loads of sex, relationships",
+                        "and health info there.",
+                        "It's also my other virtual office.",
+                    ]
+                )
+            ),
+            helper_metadata={"image": contentrepo.get_image_url("bwise_header.png")},
```

Comment:
@erikh360 just note here that the image name is just pulled out of thin air, I see the image is used a few times on the miro board but there are no images in the messages in the repo, so I just gave it a placeholder name